app/controllers/dashboard_controller.rb

```diff
@@ -216,7 +216,7 @@ def new_landing_page
 
   def browse_tag
     @tag = Tag.where(ai_text: params[:ai_text]).first
```

Comment:
I think we can use `Tag.find_by(ai_text: params[:ai_text])` in here.

Reply:
done!